test/controllers/admin/svie_controller_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+module Admin
+  class SvieControllerTest < ActionController::TestCase
```

Comment:
Something wrong with these tests, because we sign in with a wrong user.

Reply:
I've just copied it according to new module, but true, there is something wrong. I will check on it.